src/components/TextView.tsx

```diff
@@ -0,0 +1,102 @@
+import Grid from '@mui/material/Grid';
+import ChapterNavigationBar from './ChapterNavigationBar';
+import StationaryChapterNavButton from './StationaryChapterNavButton';
+import ArrowRightAltIcon from '@material-ui/icons/ArrowForward';
+import ArrowLeftAltIcon from '@material-ui/icons/ArrowBack';
+import Text from './Text';
+import react, {useEffect, useState} from 'react';
+
+
+type Note = {
+  ATTR: any,
+  _: any,
+}
+
+type w = {
+  ATTR: any,
+  note?: Note[],
+  _: string,
+}
+
+type verse = {
+  ATTR: any,
+  note?: [],
+  w: w[]
+}
+
+type ValidGreekWordNoteKeys = "OGNTsort" | "text" | "sub" | "phraseWords" | "strongs";
+
+type GreekWordNotes = {
+  OGNTSort : string,
+  strongs : string,
+  text? : string,
+  sub? : string,
+  phraseWords? : string,
+}
+
+type ValidGreekWordAttributeKeys = "lemma" | "morph";
+
+type GreekWordAttributes = {
+  [key in ValidGreekWordAttributeKeys] : string;
+}
+
+type GreekWord = {
+  notes : GreekWordNotes;
+  attributes : GreekWordAttributes;
+  text : string;
+}
+
+interface TextViewProps {
+    size: number,
+    setSize: (params: any) => any,
+    setCurrentGreekWord: (params: any) => any,
+}
+
+function TextView({size, setSize, setCurrentGreekWord}: TextViewProps)
+{
+
+  const onPhraseClick = (greekWord : GreekWord[]) =>
+  {
+    setCurrentGreekWord(greekWord);
+
+    if(size === 12)
```

Comment:
What is the purpose of this transformation?  I recommend adding a comment for tricky code like this.